perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -226,6 +250,28 @@ def ApiBasedBenchmarks(results, metadata, vm, storage, test_script_path,
       # User only wants to run the CLI based tests, do nothing here:
       return
 
+    def BuildCommandString(args):
```

Comment:
I made this function inside of ApiBasedBenchmarks() so that it could access the variables test_script_path, storage, azure_command_suffix, and host_to_connect without having to explicitly pass them as parameters. I think that makes it smaller and easier to read. However, I can certainly move it outside of ApiBasedBenchmarks() if you prefer that.

As for the name, I'll change it to something more descriptive.